docs/core/testing/unit-testing-platform-faq.md

```diff
@@ -19,3 +19,10 @@ Manually defining an entry point (`Main`) in a test project or referencing a tes
 - Disable the generation of the entry point by setting the `<GenerateTestingPlatformEntryPoint>false</GenerateTestingPlatformEntryPoint>` MSBuild property.
 
 - Completely disable the transitive dependency to `Microsoft.Testing.Platform.MSBuild` by setting the `<IsTestingPlatformApplication>false</IsTestingPlatformApplication>` MSBuild property in the project that references a test project. This is needed when you reference a test project from a non-test project, for example, a console app that references a test application.
+
+## Fakes error  Failed to resolve profiler path from COR_PROFILER_PATH and COR_PROFILER environment variables
```

Comment:
I would probably rename H1 to be platform + extensions. H2 to be on the package/feature so that L13 would be `Microsoft.Testing.Platform.MSBuild` and this one Fakes. And move the current H2 as H3